tony-history-server/startTHS

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env sh
+###########################################################################################################
+# NAME: startTHS
+#
+# DESCRIPTION:
+# This is a modified version of the `playBinary` script that was generated after `gradle dist`.
+#
+#
+# INPUT: JVM options (-Dhttp.port, -Dhttp.address, -Dconfig.resource, etc.)
+#
+#
+# OUTPUT: Logs of play.core.server.ProdServerStart
+#
+#
+# ENVIRONMENT VARIABLES:
+# JAVA_HOME: location of Java executable
+#
+#
+# NOTES: This script should only be run when it's in the distribution zip/tar.
+# Examples: after `gradle dist` from root, unzip the zip/tar in $root/build/distributions, then run:
+# ```
+# $ $root/build/distributions/startTHS -Dhttp.port=8080 -Dconfig.resource=prod.conf
```

Comment:
Looks like an extra `$` here

Reply:
Oh that was intentional. The first `$` is for showing that it's in a terminal/cmd line. The second one is to get the value of the variable `root`, which in this case is the path of the project.

Reply:
Oh. I think it would be clearer if you used
```
$ <tony-root>/build/...
```

Reply:
Oh right that looks better. Fixed: https://github.com/linkedin/TonY/pull/62/commits/9fc1d6b103ce906183051d6b89a4b75ff6c26fd9